src/pages/YXWPRedux/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import { Button, Card, Input, Table, Space } from 'antd'
+import { ShoppingHeader } from './styles/YXWPRedux.styled'
+import { PlusCircleOutlined, MinusCircleOutlined } from '@ant-design/icons'
+import { useDispatch, useSelector } from 'react-redux'
+import { increaseQuantity, decreaseQuantity, deleteItem, addItem } from '../../store/yxwp_store/actions'
+import { RootState } from '../../store/types'
+import { ShoppingListEntry } from '../../store/yxwp_store/types'
+const { Column } = Table
+
+const YXWPRedux = () => {
+  const dispatch = useDispatch()
+  const { itemList } = useSelector((state: RootState) => state.yxwp_store)
+  const blankItem: ShoppingListEntry = { itemName: '', itemQuantity: 0, key: -1 }
+  const [item, setItem] = useState<ShoppingListEntry>(blankItem)
+
+  interface Props {
+    key: number
+    itemQuantity: number
+    itemName: string
+  }
+
+  const handleAddItem = () => {
+    dispatch(addItem(item))
+    setItem(blankItem)
+  }
+
+  const updateName = (name: string) => {
+    const newItem = { ...item }
+    newItem.itemName = name
+    setItem(newItem)
+  }
+
+  const updateQuantity = (quantity: number) => {
+    const newItem = { ...item }
+    newItem.itemQuantity = quantity
+    setItem(newItem)
+  }
+
+  return (
+    <>
+      <Card>
+        <ShoppingHeader>Shopping Time! (Redux ver.)</ShoppingHeader>
+        <Space>
+          <Input placeholder="Item Name" onChange={(e) => updateName(e.target.value)} />
+          <Input placeholder="Quantity" onChange={(e) => updateQuantity(parseInt(e.target.value))} />
```

Comment:
It would seem the input boxes do not empty themselves when item has been set to blank item.

Iirc, you can do this by passing the prop value={item.itemQuantity} here. Otherwise, item is blank but it still says "4" in the input box.

Reply:
Resolved in commit b274fd2.

Now it shows blank for item name and 1 for quantity by default and after adding an item.
![image](https://user-images.githubusercontent.com/77771132/139665409-264897a9-c364-46d5-845f-5964f5bfabfe.png)